lib/buffy/throttle_and_timed.ex

```diff
@@ -348,7 +350,12 @@ defmodule Buffy.ThrottleAndTimed do
       @doc false
       @impl GenServer
       @spec handle_continue(do_work :: atom(), ThrottleAndTimed.state()) ::
-              {:noreply, ThrottleAndTimed.state()} | {:noreply, ThrottleAndTimed.state(), timeout()}
+              {:noreply,
```

Comment:
For some reason `ThrottleAndTimed.state()` doesn't work and `mix dialyzer` in `wms-service` was showing typing errors.

Reply:
I was able to fix these typing errors locally by removing the `alias` and using the full module path. If that works for you, and seems like a reasonable solution, here is a pr: https://github.com/stordco/buffy/pull/33

Reply:
Thanks, that resolved it! 👍